docs/products/cassandra/howto/disable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,113 @@
+Disable cross-cluster replication in Aiven for Apache Cassandra® |beta|
+=======================================================================
+
+This article provides you with instructions on how to discontinue the cross-cluster replication (CCR) for your Aiven for Apache Cassandra® service.
+
+About disabling CCR
+-------------------
+
+When you enable CCR for your service, you connect it to another service, which results in creating a CCR pair of services. You can disable CCR for your service either by splitting the services constituting the CCR pair or by deleting one of them.
+
+* `Aiven console <https://console.aiven.io/>`_
+* CLI
+* API
+
+It's recommended to use the Aiven console for disabling CCR.
+
+Prerequisites
+-------------
+
+* Aiven account
+* Depending on the method you choose to use for enabling CCR
+
+  * Access to the `Aiven console <https://console.aiven.io/>`_
+  * `cURL` CLI tool
+  * `Aiven CLI tool <https://github.com/aiven/aiven-client>`_
+
+* CCR enabled on a pair of Aiven for Apache Cassandra services
+
+Disable CCR in the console
+--------------------------
+
+In the console, use either of the two following methods to disable CCR on your service(s): split the services or delete one of them.
+
+Split the services
+''''''''''''''''''
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select the service for which you'd like to disable CCR.
+3. In the **Overview** tab of the service's page, navigate to **Cross Cluster Replication** and select **Split cluster**.
+
+   .. warning::
+
+      As soon as you split the cluster, the two services constituting the CCR pair become independent. It's not possible to recreate the CCR pair connecting back to the same service. To enable CCR on your service again, you can create a new service and CCR-connect it to your existing service.
```

Comment:
Should we include this warning in the About chapter as well?... on the one hand, it doesn't describe the splitting in detail, on the other hand - one might glance through the About chapter, nod and then proceed to click in the Console without ever scrolling down to see the warning :thinking: